app/main.development.js

```diff
@@ -423,14 +424,19 @@ handle("stop-dex", stopDex);
 handle("launch-dex-window", createDexWindow);
 
 function createDexWindow(serverAddress) {
-  const child = new BrowserWindow({
-    parent: mainWindow,
-    show: false,
-    autoHideMenuBar: true
+  dexWindow.loadURL("http://" + serverAddress);
+  dexWindow.once("ready-to-show", () => {
+    dexWindow.show();
   });
-  child.loadURL("http://" + serverAddress);
-  child.once("ready-to-show", () => {
-    child.show();
+  dexWindow.once("closed", () => {
+    dexWindow = new BrowserWindow({
+      show: false,
+      autoHideMenuBar: true,
+      webPreferences: {
+        nodeIntegration: true,
+        contextIsolation: false
```

Comment:
This should be the other way around (nodeIntegration: false, contextIsolation: true) for security reasons.